### PR TITLE
Conductor update

### PR DESCRIPTION
**Pull request by MitchellShiell**
## Changes

Minor updates including:

- All mentions of composer updated to conductor
- Removed Ego from the products page (in prep for overture docs update)
- Updated hero sections to be blue colour (improved contrast on product for section colours)

> [!IMPORTANT]
> This repo should be merged alongside the [conductor branch merging](https://github.com/overture-stack/composer/pull/20) and repo name change (from composer to conductor)

### CODE REVIEW

File: markdown/documentation/guides/download/clientdownload.md [hunk 1]
```diff
@@ -42,7 +41,7 @@ docker compose up --attach conductor
 
 API Keys are brokered by Keycloak and accessible when logged in to the Stage UI. For the Overture QuickStart, Stage can access from `localhost:3000`
 
-**1. Login through the Stage UI** by selecting login from the top right. Default credentials when using the Overture Composer will be username `admin` and password `admin123`.
+**1. Login through the Stage UI** by selecting login from the top right. Default credentials when using the Overture conductor will be username `admin` and password `admin123`.
```
Comment (justincorrigible) [inline markdown/documentation/guides/download/clientdownload.md:44]: should this and other mentions of the product be capitalised?

File: markdown/documentation/guides/submission/clientsubmission.md [hunk 2]
```diff
@@ -21,14 +21,13 @@ title: CLI Submissions
 **1. Clone the Quickstart repository**
 
 ```bash
-git clone  https://github.com/overture-stack/composer.git
+git clone  https://github.com/overture-stack/conductor.git
 ```
 
-**2. With Docker open, run the docker-compose**
+**2. Launch the Platform by running the appropriate command for your operating system:**
```
Comment (justincorrigible) [inline markdown/documentation/guides/submission/clientsubmission.md:27]: while on the topic of capitalisation... having this one with `P` makes it seem like a product of its own, and I think it should be `p` instead, as a regular noun.

Thoughts?

File: src/components/ProductsPageSection/index.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ export default function ProductPageSection({
   const desktopWideViewPort = isBrowser && 1215 < width && width <= 1407;
   const desktopUltraWideViewPort = isBrowser && 1408 < width;
 
-  const userDocsLink = `/documentation/${title.toLowerCase()}`;
+  const userDocsLink = `https://www.overture.bio/documentation/${title.toLowerCase()}`;
```
Comment (justincorrigible) [inline src/components/ProductsPageSection/index.js:32]: adding `www` isn't necessary anymore, opting for a simpler url pattern makes it more memorable.
```suggestion
  const userDocsLink = `https://overture.bio/documentation/${title.toLowerCase()}`;
```

File: src/pages/products/index.js [hunk 4]
```diff
@@ -45,7 +44,7 @@ export default function ProductsPage() {
         yellowButtonText1="All data submissions adhere to user-defined standards and structure"
         yellowButtonText2="Automated global identifiers tracks metadata across geographically distributed Song repositories"
         yellowButtonText3="Control the publication status of data with configurable data states"
-        isGrey={false}
+        isGrey={true}
```
Comment (justincorrigible) [inline src/pages/products/index.js:47]: in react, just havving the props means true
```suggestion
        isGrey
```

File: src/pages/products/index.js [hunk 5]
```diff
@@ -63,7 +62,7 @@ export default function ProductsPage() {
         yellowButtonText1="Support for AWS S3, Azure Storage, Google Cloud and more"
         yellowButtonText2="Including BAM and CRAM file slicing"
         yellowButtonText3="Resumable multipart uploads and downloads"
-        isGrey={true}
+        isGrey={false}
```
Comment (justincorrigible) [inline src/pages/products/index.js:65]: conversely, when properly implemented, a reusable React component should default to false on its props, so not having the props here should have the same effect
```suggestion
```

File: markdown/documentation/guides/download/clientdownload.md (hunk 1 above)
Comment (MitchellShiell) [reply markdown/documentation/guides/download/clientdownload.md:44]: Searched the repo is the only mention that is not capitalized will commit from my local branch following your review

File: markdown/documentation/guides/submission/clientsubmission.md (hunk 2 above)
Comment (MitchellShiell) [reply markdown/documentation/guides/submission/clientsubmission.md:27]: Updated

**Review by justincorrigible — APPROVED** [review]
Squasharoooooo 🦘

**Pull request merged**